### PR TITLE
feat: Update headers and special text for add database

### PR DESCRIPTION
**Pull request by hughhhh**
### SUMMARY

Update headers for creating and editing databases

### Before
Screen Shot 2021-04-30 at 11 06 04 AM

### After
Screen Shot 2021-04-30 at 11 14 35 AM

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx [hunk 1]
```diff
@@ -50,6 +52,44 @@ interface DatabaseModalProps {
 
 const DEFAULT_TAB_KEY = '1';
 
+const StyledDBModal = styled(StyledModal)`
+  .ant-alert {
+    margin-top: ${({ theme }) => theme.gridUnit * 4}px;
+    color: #325D7E;
```
Comment (michael-s-molina) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx:58]: I saw that you're using `Alert` `info`. Why are you changing the default `color`, `fontSize` and `border`? I'm worried that we'll have a lack of theme consistency.

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx [hunk 2]
```diff
@@ -50,6 +52,44 @@ interface DatabaseModalProps {
...
+    border: 1px solid #66BCFE;
+    font-size: 13px;
+    padding: 15px;
```
Comment (michael-s-molina) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx:61]:
```suggestion
    padding: ${({ theme }) => theme.gridUnit * 4}px;
```

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx [hunk 3]
```diff
@@ -50,6 +52,44 @@ interface DatabaseModalProps {
...
+    padding-top: 0;
+  }
+`;
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: ${({ theme }) => theme.gridUnit * 4}px;
+  margin-bottom: 16px;
```
Comment (michael-s-molina) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx:79]:
```suggestion
  margin-bottom: ${({ theme }) => theme.gridUnit * 4}px;
```

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx [hunk 4]
```diff
@@ -50,6 +52,44 @@ interface DatabaseModalProps {
...
+`;
+
+const HeaderTitle = styled.div`
+  color: ${({ theme }) => theme.colors.grayscale.dark1} !important;
```
Comment (michael-s-molina) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx:83]: Can you try to avoid `!important`? Maybe use a more specific class?

**Comment by michael-s-molina** [comment]
Can you also add before/after screenshots?

**Comment by hughhhh** [comment]
closing in favor of #14453

**Pull request closed without merging**